<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_read_file">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_group">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>190</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_brand">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_date1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_date2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sort">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_write_file">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter product's group:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter product's brand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter data of receipt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter execution date:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_update">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
